lib/bug_report_templates/rails_5.rb

```diff
@@ -12,132 +12,108 @@
-  # or overwrite some helpers in the base_helpers.rb file.
-  #
-  class Base < ::ApplicationController
-    # Overwrite inherit_resources to add specific InheritedResources behavior.
-    def self.inherit_resources(base)
-      base.class_eval do
-        include InheritedResources::Actions
-        include InheritedResources::BaseHelpers
-        extend  InheritedResources::ClassMethods
-        extend  InheritedResources::UrlHelpers
-
-        # Add at least :html mime type
-        respond_to :html if self.mimes_for_respond_to.empty?
-        self.responder = InheritedResources::Responder
-
-        helper_method :resource, :collection, :resource_class, :association_chain,
-                      :resource_instance_name, :resource_collection_name,
-                      :resource_url, :resource_path,
-                      :collection_url, :collection_path,
-                      :new_resource_url, :new_resource_path,
-                      :edit_resource_url, :edit_resource_path,
-                      :parent_url, :parent_path,
-                      :smart_resource_url, :smart_collection_url
-
-        self.class_attribute :resource_class, :instance_writer => false unless self.respond_to? :resource_class
-        self.class_attribute :parents_symbols,  :resources_configuration, :instance_writer => false
-
-        protected :resource_class, :parents_symbols, :resources_configuration,
-          :resource_class?, :parents_symbols?, :resources_configuration?
-      end
-    end
-
-    inherit_resources(self)
-  end
+# create ApplicationController that needed for inherited_resources
+class ApplicationController < ActionController::Base
 end
 
-require 'activeadmin'
+# initialize app
+require 'action_controller/railtie'
+require 'action_view/railtie'
 
 class TestApp < Rails::Application
   config.root = File.dirname(__FILE__)
-  config.logger = Logger.new($stdout)
-  Rails.logger  = config.logger
+  config.logger = Logger.new(STDOUT)
+  Rails.logger = config.logger
 
-  secrets.secret_token    = 'secret_token'
+  secrets.secret_token = 'secret_token'
   secrets.secret_key_base = 'secret_key_base'
 
-  # Define any models required to duplicate your issue
-  class YourModel < ActiveRecord::Base; end;
+  config.eager_load = false
+end
 
-  # Define any ActiveAdmin configuration necessary to duplicate your issue
-  ActiveAdmin.setup do |config|
-    # Authentication disabled by default. Override if necessary
-    config.authentication_method = false
-    config.current_user_method   = false
-  end
+Rails.application.initialize!
 
-  ActiveAdmin.register_page 'Dashboard' do
-    content do
-      'Test Me'
-    end
-  end
+# create models
+class YourModel < ActiveRecord::Base
+end
 
-  # Set up your models as you have them in your ActiveAdmin config
-  ActiveAdmin.register YourModel do
-  end
+# configure active_admin
+ActiveAdmin.application.authentication_method = false
+ActiveAdmin.application.current_user_method = false
```

Comment:
I prefer the original configuration via a `setup` block. It's very likely that the user will customize it so it seems handy not having to specify `ActiveAdmin.application` for each configuration she wants to add.